_data/locales/en.yml

```diff
@@ -401,8 +401,8 @@ leap_ovw_desc: Leap uses source from SUSE Linux Enterprise (SLE), which gives Le
 tumbleweed_ovw_title: State-of-the-art desktop and server operating system
 tumbleweed_ovw_desc: With Tumbleweed you don't have to take difficult decisions about things you value, either freedom or safety, either control or security, technology or stability -- Tumbleweed lets you have your cake and eat it too!
 leap_micro_ovw_title: Leap Micro is an ultra-reliable, lightweight operating system built for containerized and virtualized workloads.
-leap_micro_ovw_desc_p1: This community version is based on SUSE Linux Enterprise Micro, which leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise. This merging of technologies provides for a modern, immutable and developer-friendly OS platform.
-leap_micro_ovw_desc_p2: Leap Micro offers an offline image. The main difference between the offline and self-install/raw images are that the offline image has an installer. Raw and self-install allows for customization via combustion or manually in the image after it is written to the disk. There is an option for a real-time kernel.
+leap_micro_ovw_desc_p1: This community version is based on SUSE Linux Micro, which leverages the enterprise-hardened security and compliance components of SUSE Linux Enterprise. This merging of technologies provides for a modern, immutable, and developer-friendly OS platform.
```

Comment:
SUSE _officially_ still refers to it as "SUSE Linux Enterprise Micro": https://www.suse.com/de-de/products/micro/. I think it should align.